mysql-test/t/slow_log_legacy_user.test

```diff
@@ -0,0 +1,71 @@
+#
+# Test log_legacy_user
+#
+
+--source include/have_innodb.inc
+
+# Just in case slow_log_wait_time-slow.log already exists, delete it first.
+--perl
```

Comment:
If you wanted, I think you could simplify this by removing the perl script and using this:

```
--remove_files_wildcard $MYSQLTEST_VARDIR/mysqld.1/data slow_log_wait_time-slow.log
```